packages/lib/delegationCredential/server.ts

```diff
@@ -188,7 +188,11 @@ const _buildDelegatedConferencingCredential = ({
 /**
  * Gets calendar as well as conferencing credentials(stored in-memory) for the user from the corresponding enabled DelegationCredential.
  */
-export async function getAllDelegationCredentialsForUser({ user }: { user: { email: string; id: number } }) {
+export async function getAllDelegationCredentialsForUserIncludeServiceAccountKey({
+  user,
+}: {
+  user: { email: string; id: number };
+}) {
   log.debug("called with", safeStringify({ user }));
```

Comment:
Are we sure user doesn't contain credentials?

Reply:
That's just debug mode, so didn't focus on it. Can clean those up later